create-jira-release-ticket/action.yml

```diff
@@ -88,27 +81,48 @@ runs:
       env:
         JIRA_USER: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_USER }}
         JIRA_TOKEN: ${{ fromJSON(steps.secrets.outputs.vault).JIRA_TOKEN }}
+        JIRA_PROD_URL: "https://sonarsource.atlassian.net/"
+        JIRA_SANDBOX_URL: "https://sonarsource-sandbox-608.atlassian.net/"
       run: |
-        SANDBOX_FLAG=""
-        if [[ "${{ inputs.use_sandbox }}" == "true" ]]; then
-          SANDBOX_FLAG="--use-sandbox"
+        # Determine project key from input or environment variable
+        PROJECT_KEY="${{ inputs.jira-project-key }}"
+        if [[ -z "$PROJECT_KEY" ]]; then
+          PROJECT_KEY="${JIRA_PROJECT_KEY:-}"
+        fi
```

Comment:
Same as in the other PR, I am more in favor of the `${{ inputs.jira-project-key || env.JIRA_PROJECT_KEY }}` syntax